File: twoSum.cpp
#include <vector>
#include <map>

using namespace std;

class Solution
{
    public:
    vector<int> twoSum(vector<int>& nums, int target)
    {
        vector<int> num;
        map<int,int> map;
        
        for(int i = 0; i < nums.size(); i++)
        {
            int diff = target - nums[i];
            
            if(map.count(diff))
            {
                num.push_back(map.at(diff));
                num.push_back(i);
            
                return num;
            }
            
            map.insert(pair<int,int>(nums[i],i));
        }
        return num;
    }
};

int main()
{

    return 0;
}

File: twoSum.py
# Function to find indices
def twoSum(nums, target):
   # Finding list size
   list_size= len(nums)
  
   # Initialize empty array
   result=[]
  
   # Loop to iterate over list
   for i in range(0,list_size-1):
       for j in range(i,list_size):
           if(nums[i]+nums[j]==target):
               result.append(i)
               result.append(j)
               # return list with indices
               return result
  
   # Return empty list if sum not found
   return result

# list initialize
nums= [2,7,11,15]

# Target Initialize
target= 9

# Output
print("The output is: ")
print(twoSum(nums, target))
